system/services/src/com/percussion/services/assembly/impl/nav/PSNavHelper.java

```diff
@@ -881,7 +881,7 @@ public PSLocator findNavon(PSLocator folderLoc)
          return null;
       }
 
-      if (folderLoc!= null && folderLoc.getId()==1) {
+      if (folderLoc.getId() == 1) {
```

Comment:
Can throw an NPE now...

Reply:
It is impossible for folderLoc to be null at that location.  line 877 checks for null and exits the function.